src/Apps/Collect2/Routes/Collect/index.tsx

```diff
@@ -100,7 +101,11 @@ export const CollectApp = track({
               const url = buildUrlForCollectApp(filters)
 
               if (typeof window !== "undefined") {
-                window.history.replaceState({}, "", url)
+                // FIXME: Is this the best way to guard against history updates
+                // in Storybooks?
+                if (!process.env.IS_STORYBOOK) {
+                  window.history.replaceState({}, "", url)
+                }
```

Comment:
?

Reply:
Haven't been able to think through an alternative here, but it's related to when we develop in Storybooks, which loads things inside of an iframe. It's a bit hacky but we can leave for now.

Reply:
On the other hand, while it's awesome to be able to load the app shell in storybooks, _and_ be able to browse around among pages...at some point it becomes no longer realistic to use storybooks for that type of development.

I pretty much only ran this linked in Force (vs. storybooks), b/c of so much other underlying code changes. Didn't run into any weird issues, that dev flow is solid.

So while storybooks is great for isolated things/UI, getting navigation working, plus anything SSR related, is bit of a stretch.